commitizen/formats/errors.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import annotations
+
+
+class FormatError(Exception):
```

Comment:
It would be better if we could inherit from https://github.com/commitizen-tools/commitizen/blob/master/commitizen/exceptions.py#L38 and assign an error code to it

Reply:
Done 👍🏼 
The `commitizen.changelog_format.errors` has been removed and `ChangelogFormatUnknown` moved into `commitizen.exceptions` (only exception really used) and `exit_code` has been added (doc was not up to date on those, so I also updated it)